graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/exceptions/DefaultDataFetcherExceptionHandler.kt

```diff
@@ -23,18 +23,29 @@ import graphql.execution.DataFetcherExceptionHandlerResult
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 import org.springframework.util.ClassUtils
+import java.util.concurrent.CompletableFuture
 
 /**
  * Default DataFetcherExceptionHandler used by the framework, can be replaced with a custom implementation.
  * The default implementation uses the Common Errors library to return GraphQL errors.
  */
 class DefaultDataFetcherExceptionHandler : DataFetcherExceptionHandler {
 
-    override fun onException(handlerParameters: DataFetcherExceptionHandlerParameters?): DataFetcherExceptionHandlerResult {
+    @Deprecated("Deprecated in GraphQL Java", replaceWith = ReplaceWith("handlerException(handlerParameters)"))
```

Comment:
handlerException => handleException

Reply:
Thanks, such a small change and I can't avoid a typo :P